src/app/website/src/team.ml

```diff
@@ -129,6 +129,39 @@ let core () =
        NICTA. He also worked on the initial implementation of Alacris, a \
        cryptocurrency solution layered on top of existing blockchains. He \
        holds a PhD in computer science from Northeastern University."
+  and vanishree =
+    plain "Vanishree Rao" "Protocol Researcher"
+      "Vanishree is a theoretical and applied cryptographer with deep \
+       experience in industry and academia. She earned her PhD at UCLA \
+       through her work on zk-proofs, multiparty computation, hashing, and \
+       pseudorandom functions, among other projects. She then worked in \
+       industry at Xerox PARC and Intertrust Technologies. Vanishree enjoys \
+       developing cryptographic solutions for real-world challenges and \
+       communicating intuitive explanations of complex cryptography concepts."
+  and echo =
+    plain "Echo Nolan" "Protocol Engineer"
+      "Echo is interested in programming languages, type systems and \
+       prediction markets. He has made open source contributions to Idris and \
+       various parts of the Haskell ecosystem. He's also made and lost a fair \
+       amount of money trading predictions on Augur. Before joining O(1) \
+       Labs, Echo worked on an text messaging platform for educators, using \
```

Comment:
Oops, didn't catch this when Brad sent me his edit. The users of Signal Vine's platform are academic counselors, not actually "educators".